Mobile/ShopLens/Droid/Properties/AndroidManifest.xml

```diff
@@ -3,15 +3,10 @@
 	<uses-sdk android:minSdkVersion="10" android:targetSdkVersion="27" />
 	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 	<uses-permission android:name="android.permission.RECORD_AUDIO" />
+	<uses-permission android:name="android.permission.CAMERA" />
 	<application android:label="ShopLens">
-		<provider 
-      android:name="android.support.v4.content.FileProvider" 
-      android:authorities="${applicationId}.shoplens.fileprovider" 
-      android:exported="false" 
-      android:grantUriPermissions="true">
-			<meta-data 
-        android:name="android.support.FILE_PROVIDER_PATHS" 
-        android:resource="@xml/provider_paths" />
+		<provider android:name="android.support.v4.content.FileProvider" android:authorities="${applicationId}.shoplens.fileprovider" android:exported="false" android:grantUriPermissions="true">
```

Comment:
Oh so the contents of the tags should all be in one line? Is that what you meant when you said you have to align the axml files?

Reply:
I don't know about this one, personally I liked it better as separate lines

Reply:
This file's style is auto generated, since I'm editing it NOT from the Source but using checkboxes.